cmd/api-errors.go

```diff
@@ -1500,6 +1506,8 @@ func toAPIErrorCode(ctx context.Context, err error) (apiErr APIErrorCode) {
 		apiErr = ErrInvalidRange
 	case errDataTooLarge:
 		apiErr = ErrEntityTooLarge
+	case errPolicyTooLarge:
```

Comment:
You don't need this change in this PR since this error is not used anywhere..

Reply:
Agree, deleted